ImperatorToCK3/Data_Files/configurables/nickname_map.txt

```diff
@@ -31,50 +31,50 @@ link = { imp = NICKNAME_CURIO imp = NICKNAME_THE_DEVOUT ck3 = nick_the_pious }
 link = { imp = NICKNAME_THE_PARAGON ck3 = nick_the_shepherd }
 link = { imp = NICKNAME_LUPUS imp = NICKNAME_THE_WOLF ck3 = nick_the_wolf }
 link = { imp = NICKNAME_MARIUS ck3 = nick_the_overseer }
-link = { imp = NICKNAME_BRITANNICUS imp = NICKNAME_THE_BRITON ck3 = nick_the_lionheart }
-#link = { imp = NICKNAME_GALLICUS imp = NICKNAME_THE_GAUL ck3 = ??? } #"The French"
-#link = { imp = NICKNAME_BELGICUS imp = NICKNAME_THE_BELGAE ck3 = ??? } #"The Belgian"
+link = { imp = NICKNAME_BRITANNICUS imp = NICKNAME_THE_BRITON ck3 = nick_the_british }
+link = { imp = NICKNAME_GALLICUS imp = NICKNAME_THE_GAUL ck3 = nick_the_french }
+link = { imp = NICKNAME_BELGICUS imp = NICKNAME_THE_BELGAE ck3 = nick_the_belgian }
 link = { imp = NICKNAME_GERMANICUS imp = NICKNAME_THE_GERMAN ck3 = nick_the_german }
-#link = { imp = NICKNAME_RHAETICUS imp = NICKNAME_THE_RHAETIAN ck3 = ??? } #"The Rhetian"
-#link = { imp = NICKNAME_ITALICUS imp = NICKNAME_THE_ITALIOTE ck3 = ??? } #'The Italiote'
-#link = { imp = NICKNAME_ILLYRICUS imp = NICKNAME_THE_ILLYRIAN ck3 = ??? } #'The Illyrian'
+link = { imp = NICKNAME_RHAETICUS imp = NICKNAME_THE_RHAETIAN ck3 = nick_the_swiss }
+link = { imp = NICKNAME_ITALICUS imp = NICKNAME_THE_ITALIOTE ck3 = nick_the_italian }
+link = { imp = NICKNAME_ILLYRICUS imp = NICKNAME_THE_ILLYRIAN ck3 = nick_the_dalmatian }
 link = { imp = NICKNAME_HISPANICUS imp = NICKNAME_THE_IBERIAN ck3 = nick_the_motherfather_of_spain }
-#link = { imp = NICKNAME_MAURETANICUS imp = NICKNAME_THE_MAURETANIAN ck3 = ??? } #'The Mauretanian'
-#link = { imp = NICKNAME_NUMIDIUS imp = NICKNAME_THE_NUMIDIAN ck3 = ??? } #'The Numidian'
+link = { imp = NICKNAME_MAURETANICUS imp = NICKNAME_THE_MAURETANIAN ck3 = nick_the_moor }
+link = { imp = NICKNAME_NUMIDIUS imp = NICKNAME_THE_NUMIDIAN ck3 = nick_the_algerian }
 link = { imp = NICKNAME_VENEDICUS imp = NICKNAME_THE_VENEDIAN ck3 = nick_the_wend }
-#link = { imp = NICKNAME_DACIUS imp = NICKNAME_THE_DACIAN ck3 = ??? } #'The Dacian'
-#link = { imp = NICKNAME_HELLENICUS imp = NICKNAME_THE_HELLENE ck3 = ??? } #'The Hellene'
-#link = { imp = NICKNAME_CYRENAICUS imp = NICKNAME_THE_CYRENE ck3 = ??? } #'The Cyrene'
-#link = { imp = NICKNAME_AEGYPTUS imp = NICKNAME_THE_EGYPTIAN ck3 = ??? } #'The Egyptian'
+link = { imp = NICKNAME_DACIUS imp = NICKNAME_THE_DACIAN ck3 = nick_the_romanian }
```

Comment:
Why not just Dacian? Only a Romanized Dacian would be called Romanian..

Reply:
Ditto